### ISSUE TITLE
General issue: console.log

### ISSUE THREAD

**Issue by michaelKurowski** [Jordan141/YelpCamp#27]
You use `console.log` to log errors. Use tools designed for such practice, preferably with ability to dump errors into files. `console.log` should never end up in production code.

**Comment by Jordan141** [comment]
Yup that's the next refactor I'm doing, meaningful error handling, should be completed by the end of today.
